.github/scripts/upload-scribe.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# [FB Internal] Upload every json file to Scribe
+
+# Use the config v0
+CONFIG_FILE=/config/config-v0-notacotron-noslomo.json
```

Comment:
i think it'd be good to delete the notacotron-noslomo part from the name.

the reason being, i think going forward any score version will include some subset of the models that are checked in, and we can keep the names simple.  

on the same token though, could you add a markdown document to describe the v0 score?  i think we should have a doc that shows what is supported for each score version, and explains the theory about it.  i can help with that if you want (can be a separate PR)

Reply:
oh and the doc should also describe the machine type and any other assumptions about how it's run and configured.